test/math.js

```diff
@@ -551,7 +565,11 @@ describe('Math', function () {
       expect(Math.sinh(-Infinity)).to.equal(-Infinity);
       expect(Math.sinh(-5)).to.almostEqual(-74.20321057778875);
       expect(Math.sinh(2)).to.almostEqual(3.6268604078470186);
-      expect(Math.sinh(-2e-17)).to.equal(-2e-17);
+      expect(Math.sinh(-2e-17)).to.almostEqual(-2e-17);
```

Comment:
@ljharb in a browser of 2150 year, which have Number.EPSILON < 2e-17 * 2e-17 / 3